layers/+lang/javascript/packages.el

```diff
@@ -195,6 +196,24 @@
         "sR" 'spacemacs/skewer-eval-region-and-focus
         "ss" 'skewer-repl))))
 
+(defun javascript/init-nodejs-repl ()
+  (use-package nodejs-repl
+    :defer nil
+    :init
+    (spacemacs/register-repl 'nodejs-repl
+                              'nodejs-repl
+                              "nodejs-repl")
+    :config
+    (progn
+      (spacemacs/declare-prefix-for-mode 'js2-mode "mn" "nodejs")
+      (spacemacs/set-leader-keys-for-major-mode 'js2-mode
```

Comment:
Please follow the [spacemacs conventions for REPLs](http://develop.spacemacs.org/doc/CONVENTIONS.html#repls)